browser/src/control/jsdialog/Widget.TreeView.js

```diff
@@ -174,6 +174,9 @@ function _treelistboxEntry(parentContainer, treeViewData, entry, builder, isTree
 	var disabled = treeViewData.enabled === 'false' || treeViewData.enabled === false;
 
 	var li = L.DomUtil.create('li', builder.options.cssClass, parentContainer);
+	if (entry.text === 'Separator') {
```

Comment:
Please add function `isSeparator` so it will be in one place.
There is no other property like `type` or similar? Do we need to detect by test?

Reply:
>Please add function isSeparator so it will be in one place.

Sure i will do that 

>There is no other property like type or similar?

No, it does not have any type property for separator 

```
entry : {
    columns: [{text: 'Separator'}]
    row: 4
    text: "Separator"
  }
```
 
 >Do we need to detect by test?
 
 Yes we need to add test for detect separator is handled in tree widget view